tegra-eeprom-tool.nix

```diff
@@ -1,31 +1,16 @@
-{ stdenv, lib, fetchFromGitHub, fetchpatch, cmake, pkg-config, libedit }:
+{ stdenv, lib, fetchFromGitHub, cmake, pkg-config, libedit }:
 
 stdenv.mkDerivation rec {
   pname = "tegra-eeprom-tool";
-  version = "v2.0.1";
+  version = "283a701d25283a4e5584f5fe072bcb2d7d2ae1b1";
```

Comment:
Could you instead set the version to "unstable-YYYY-MM-DD", to match upstream nixpkgs policy?